src/driver/Driver.ts

```diff
@@ -47,6 +47,11 @@ export interface Driver {
     dataTypeDefaults: DataTypeDefaults;
 
     /**
+     * Gets list of column data types that support length by a driver.
+     */
+    withLengthColumnType: ColumnType[];
```

Comment:
this should be called `withLengthColumnTypes` since its an array.

Reply:
Also I don't really like `withLengthColumnType` name but I cannot suggest something better at the moment

Reply:
Sorry for the typo, I fixed it